src/components/Pagination/Pagination.tsx

```diff
@@ -178,7 +178,9 @@ export const Pagination = ({
   const secondPage = decideSecondPage({ page, pages });
   const thirdPage = decideThirdPage({ page, pages });
   const forthPage = decideForthPage({ page, pages });
-  const showLastPage = page === pages - 2;
+  const showThirdPage = pages > 2;
```

Comment:
Took me a second to reason with `pages` being `numberOfPages` or `totalPages`, maybe it could be slightly more descriptive?